.kube-linter.yaml

```diff
@@ -0,0 +1,3 @@
+checks:
+  ignorePaths:
+    - components/build-service/components/rh-certs/add-rh-certs-patch.yaml
```

Comment:
How does the add-certs patch fail linting?

Reply:
because it doesn't contain a full definition of a deployment (and this is ok)

Reply:
ok, cool. I ran kube-linter on the entire directory (not just on the output of kustomize build). I'll remove the config file.

Reply:
done